models/iaf_psc_alpha_canon.h

```diff
@@ -86,6 +86,9 @@ dynamics are integrated using exact integration between events [2]_.
    A further improvement of precise simulation is implemented in
    iaf_psc_exp_ps based on [3]_.
 
+For implementation details see the
```

Comment:
This text is repeated below (under "Remarks"). Suggest to unify as a ``.. note`` like in the other models.